dev/crypto-notes.md

```diff
@@ -0,0 +1,35 @@
+## Important notes
+
+### Key-pair generation
+
```

Comment:
add: `it is critical that we check cryptographic operations to ensure the right amount of entropy is being sourced`.